### PR TITLE
refactor aaparser pkg, add unit tests

### PR DESCRIPTION
**Pull request by jessfraz**
Signed-off-by: Jessica Frazelle acidburn@docker.com

### CODE REVIEW

File: pkg/aaparser/aaparser.go [hunk 1]
```diff
@@ -1,35 +1,81 @@
...
+	if err != nil {
+		return -1, -1, err
+	}
+
+	return parseVersion(string(output))
+}
+
+// IsEnabled returns true if apparmor is enabled for the host.
+func IsEnabled() bool {
+	if _, err := os.Stat("/sys/kernel/security/apparmor"); err == nil {
```
Comment (thaJeztah) [inline pkg/aaparser/aaparser.go]: would it be cleaner to use early returns here, in stead of nested ifs?

**Comment by thaJeztah** [comment]
one nit/suggestion, but LGTM

File: pkg/aaparser/aaparser.go (hunk 1 above)
Comment (jfrazelle) [inline pkg/aaparser/aaparser.go]: oh we actually dont need that function, was using this pkg elsewhere, will update

**Comment by jfrazelle** [comment]
easy merge for whoever wants it

**Comment by estesp** [comment]
LGTM

**Pull request merged**